sbpy/photometry/core.py

```diff
@@ -5,13 +5,255 @@
 created on June 23, 2017
 """
 
-__all__ = ['DiskIntegratedModelClass', 'HG', 'HG12', 'HG1G2',
-           'DiskFunctionModel', 'LommelSeeliger', 'Lambert',
+__all__ = ['ref2mag', 'mag2ref', 'spline',
+           'DiskIntegratedModelClass', 'HG', 'HG12', 'HG1G2',
+           'DiskFunctionModel', 'LommelSeeliger', 'Lambert', 'LunarLambert',
            'PhaseFunctionModel', 'ROLOPhase',
            'ResolvedPhotometricModelClass', 'ROLO']
 
+import numpy as np
+from scipy.integrate import quad
+from astropy.modeling import FittableModel, Fittable1DModel, Fittable2DModel, Parameter
+from astropy import units
+
+
+def ref2mag(ref, radius, M_sun=None):
+    """Convert average bidirectional reflectance to reduced magnitude
+
+    Parameters
+    ----------
+    ref : num, astropy.units.Quantity
+        Average bidirectional reflectance
+    radius : num, astropy.units.Quantity
+        Radius of object
+    M_sun : number, optional
+        The magnitude of the Sun, default is -26.74
+
+    Returns
+    -------
+    The same type as input, reduced magnitude (at r=delta=1 au)
+
+    Examples
+    --------
+    >>> from astropy import units as u
```

Comment:
I'm not sure what you mean here.  Could you explain?  Thanks.

Reply:
The examples that you currently provide for all the functions do not `import` (sorry, I meant `import` not `include`; been doing some C++ recently...) your photometry module. It would be good to include this so that people know how to access the functions, just so that the examples are complete.

Reply:
Got it.  Was confused by `include`.  Will do.